email/sender.go

```diff
@@ -27,14 +27,19 @@ func (s SMTPSender) Send(from string, to []string, msg []byte) error {
 
 // NewSMTPSender returns a new SMTPSender according to the provided mail configuration
 func NewSMTPSender(cfg *config.Mail) *SMTPSender {
-	auth := smtp.PlainAuth(
-		"",
-		cfg.User,
-		cfg.Password,
-		cfg.Host,
-	)
-	if cfg.Host == "localhost" {
-		auth = unencryptedAuth{auth}
+	var auth smtp.Auth
+	if cfg.Encrypted {
+		auth = smtp.PlainAuth(
+			"",
+			cfg.User,
+			cfg.Password,
+			cfg.Host,
+		)
+		if cfg.Host == "localhost" {
+			auth = unencryptedAuth{auth}
+		}
+	} else {
+		auth = smtp.CRAMMD5Auth(cfg.User, cfg.Password)
 	}
```

Comment:
Is this necessary? wouldn't the following be simpler
```suggestion
	var auth smtp.Auth
	auth = smtp.PlainAuth(
			"",
			cfg.User,
			cfg.Password,
			cfg.Host,
		)
	if !cfg.Encrypted {
	      auth = unencryptedAuth{auth}
	} 
```

Reply:
This also throws unencrypted connection, PlainAuth will only send the credentials if the connection is using TLS or is connected to localhost. 

We connect to mailhog in the container [here](https://github.com/ONSdigital/dp-compose/blob/e8d5fcb34e120031c199cf31a8e4bd26ad7b6fad/v2/manifests/core-ons/dp-feedback-api.yml#L25)

Reply:
Sure, makes sense - is there a solution that will work for both localhost and in docker?

Reply:
Yes, Encrypted would only be false on docker, then we will use CRAMMD5Auth instead of PlainAuth

Reply:
```suggestion
	var auth smtp.Auth
	if cfg.Encrypted {
		auth = smtp.PlainAuth(
			"",
			cfg.User,
			cfg.Password,
			cfg.Host,
		)
	} else {
		auth = smtp.CRAMMD5Auth(cfg.User, cfg.Password)
	}
```

Why not this then?

Reply:
Done